src/test/java/org/kainos/ea/integration/LoginIntegrationTest.java

```diff
@@ -0,0 +1,39 @@
+package org.kainos.ea.integration;
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.DropwizardWebServiceApplication;
+import org.kainos.ea.DropwizardWebServiceConfiguration;
+import org.kainos.ea.cli.Login;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class LoginIntegrationTest {
+
+    static final DropwizardAppExtension<DropwizardWebServiceConfiguration> APP = new DropwizardAppExtension<>(
+            DropwizardWebServiceApplication.class, null,
+            new ResourceConfigurationSourceProvider()
+    );
+    Login validLoginRequest = new Login("admin@admin.com", "securePassword");
```

Comment:
If this is a valid login to the database, it's not very secure to have the credentials available to the public in the source code